README.md

```diff
@@ -0,0 +1,49 @@
+# Integreat Compass
+
+This system gathers courses that job center employees can pass on in counseling sessions to refugees and newcomers.
```

Comment:
```suggestion
This is a Django 4 based web application, a project powered by the Tür an Tür [Digitalfabrik](https://tuerantuer.de/digitalfabrik/). The main goal is to develop an application that gathers courses that job center employees can pass on in counseling sessions to refugees and newcomers.
```
I like the opening line we have for [Lunes CMS](https://github.com/digitalfabrik/lunes-cms), although I'm aware that web application might not be the best description. I like it, because it also introduces the technology and the main contributors right away